LICENSE

```diff
@@ -0,0 +1,21 @@
+The MIT License (MIT)
+
+Copyright (c) 2016 Office for National Statistics
```

Comment:
This also needs updating across repos. I'll add a card to Trello for the other services.